rfcs/formats/network/iin_method.md

```diff
@@ -0,0 +1,264 @@
+    "id": "did:iin:<iin_name>:<network_name>",
+    "networkParticipants": [
+      "did:iin:<iin_name>:<network_participant_1>",
+      "did:iin:<iin_name>:<network_participant_2>",
+      "did:iin:<iin_name>:<network_participant_3>"
+    ],
+    "verificationMethod": [{
+        "id": "did:iin:<iin_name>:<network_name>#multisig",
+        "type": "GroupMultiSig",
+        "controller": "did:iin:<iin_name>:<network_name>",
+        "multisigKeys": [
+          "did:iin:<iin_name>:<network_participant_1>#key1",
+          "did:iin:<iin_name>:<network_participant_2>#key3",
+          "did:iin:<iin_name>:<network_participant_3>#key1"
+        ],
+        "updatePolicy": {
+          "id": "did:iin:<iin_name>:<network_name>#updatepolicy",
+          "controller": "did:iin:<iin_name>:<network_name>",
+          "type": "VerifiableCondition2021",
+          "conditionAnd": [{
+              "id": "did:iin:<iin_name>:<network_name>#updatepolicy-1",
+              "controller": "did:iin:<iin_name>:<network_name>",
+              "type": "VerifiableCondition2021",
+              "conditionOr": ["did:iin:<iin_name>:<network_participant_3>#key1",
+                "did:iin:<iin_name>:<network_participant_2>#key3"
+              ]
+            },
+            "did:iin:<iin_name>:<network_participant_1>#key1"
+          ]
+        }
+      },
+
+      {
+        "id": "did:iin:<iin_name>:<network_name>#fabriccerts",
+        "type": "DataplaneCredentials",
+        "controller": "did:iin:<iin_name>:<network_name>",
+        "FabricCredentials": {
+          "did:iin:<iin_name>:<network_participant_1>": "Certificate3_Hash",
+          "did:iin:<iin_name>:<network_participant_2>": "Certificate2_Hash",
+          "did:iin:<iin_name>:<network_participant_3>": "Certificate3_Hash"
+        }
+      }
+    ],
+    "authentication": [
+      "did:iin:<iin_name>:<network_name>#multisig"
+    ],
+    "relayEndpoints": [{
+        "hostname": "10.0.0.8",
+        "port": "8888"
+      },
+      {
+        "hostname": "10.0.0.9",
+        "port": "8888"
+      }
+
+    ]
+  },
+  "signatures": {
+    "did:iin:<iin_name>:<network_participant_1>": "...",
+    "did:iin:<iin_name>:<network_participant_2>": "...",
+    "did:iin:<iin_name>:<network_participant_3>": "..."
+  }
+}
+
+```
+
+How this request is created is out of the scopt of this specification. However, generally it is recommended that the signatures of the request are collected through a smart contract in the network blockchain itself.
+
+#####  Validation of Network DID creation request by the IIN
+
+The DID create method of the IIN registry must authenticate the a Network DID creation request differently from other DIDs. For a Network DID, the IIN first validates if the DID Document has the requisite properties: `networkParticipants`, and a verificationMethod of type `GroupMultiSig`. Additionally, the request must contain the signatures from all participants of the network authenticating it.
```

Comment:
"create" in quotes?
"the a" --> "a"
Remove "For a network DID,"
"has the requisite properties: `networkParticipants`, and a verificationMethod of type `GroupMultiSig`" --> "has the `networkParticipants` property and the `verificationMethod` property of value `GroupMultiSig`"
"network authenticating it": I'm not sure I understand this.